pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/sqlinstance-locationpreference/update.yaml

```diff
@@ -22,6 +22,7 @@ spec:
   databaseVersion: POSTGRES_16
   region: us-central1
   settings:
+    availabilityType: REGIONAL
```

Comment:
Is this an immutable field?

Reply:
It is actually mutable. I just updated the test case to cover mutating this field also.

Reply:
Good to know! Normally location has more constraints. Have you run against real GCP?

Reply:
Yes.